features/nationaliteithistorie/dev/nationaliteit-gegevens.feature

```diff
@@ -0,0 +1,129 @@
+# language: nl
+
+Functionaliteit: Lever de juiste gegevens over een nationaliteit
+
+  Als consumer van de BRP
+  Wil ik de nationaliteiten van een persoon met alle geldige gegevens
+
+  In de bron worden gegevens over een nationaliteit of bijzonder Nederlanderschap opgeslagen in één categorievoorkomen met de actuele gegevens (categorie 4) en optioneel in een of meerdere categorievoorkomens met "historische" gegevens (categorie 54).
```

Comment:
Ik weet niet of het duidelijk is hoe de actuele en historische gegevens in de Gegeven stappen wordt beschrijven. Is het niet beter om aan te geven dat in de tabel van een Gegeven stap de eerste rij de actuele gegevens bevat en de andere rijen de historische gegevens?

Reply:
Dat kan, maar dan moet je in de Gegeven stappen bij meerdere nationaliteiten (met eventuele historie) per nationaliteit een aparte Gegeven stap maken. 
Het lijkt mij ook goed om dit dan te beschrijven in de feature zelf (een soort readme aan het begin van de feature.

Reply:
Ik denk dat je bij meerdere nationaliteiten inderdaad voor elke nationaliteit een aparte Gegeven stap moet maken. Anders is niet duidelijk dat het een andere nationaliteit betreft en niet een wijziging/correctie op dezelfde nationaliteit

Reply:
Ik zal in de features een toelichting toevoegen over de volgorde in de Gegeven stappen